app/javascript/components/server-components/EmailsPage/PublishedTab.tsx

```diff
@@ -133,39 +124,32 @@ export const PublishedTab = () => {
                     onClick={() => setSelectedInstallmentId(installment.external_id)}
                   >
                     <td data-label="Subject">{installment.name}</td>
-                    <td data-label="Date" style={{ whiteSpace: "nowrap" }}>
+                    <td data-label="Date" className="whitespace-nowrap">
                       {new Date(installment.published_at).toLocaleDateString(userAgentInfo.locale, {
                         day: "numeric",
                         month: "short",
                         year: "numeric",
                         timeZone: currentSeller.timeZone.name,
                       })}
                     </td>
-                    <td data-label="Emailed" style={{ whiteSpace: "nowrap" }}>
+                    <td data-label="Emailed" className="whitespace-nowrap">
                       {installment.send_emails ? formatStatNumber({ value: installment.sent_count }) : "n/a"}
                     </td>
-                    <td data-label="Opened" style={{ whiteSpace: "nowrap" }}>
+                    <td data-label="Opened" className="whitespace-nowrap">
                       {installment.send_emails
                         ? formatStatNumber({ value: installment.open_rate, suffix: "%" })
                         : "n/a"}
                     </td>
-                    <td data-label="Clicks" style={{ whiteSpace: "nowrap" }}>
+                    <td data-label="Clicks" className="whitespace-nowrap">
                       {installment.clicked_urls.length > 0 ? (
                         <span className="has-tooltip" aria-describedby={`url-clicks-${installment.external_id}`}>
                           {formatStatNumber({ value: installment.click_count })}
-                          <div
-                            role="tooltip"
-                            id={`url-clicks-${installment.external_id}`}
-                            style={{ padding: 0, width: "20rem" }}
-                          >
+                          <div role="tooltip" id={`url-clicks-${installment.external_id}`} className="w-[20rem] p-0">
                             <table>
                               <tbody>
                                 {installment.clicked_urls.map(({ url, count }) => (
                                   <tr key={`${installment.external_id}-${url}`}>
-                                    <th
-                                      scope="row"
-                                      style={{ whiteSpace: "break-spaces", maxWidth: "calc(20rem * 0.7)" }}
-                                    >
+                                    <th scope="row" className="max-w-[calc(20rem*0.7)] whitespace-break-spaces">
```

Comment:
(leaving this as is, it's a bit odd but we can tidy things like this up once we upgrade to Tailwind 4 and can use arbitrary numbers for sizing)